uui/components/layout/CheckboxGroup.tsx

```diff
@@ -1,10 +1,76 @@
-import { withMods } from '@epam/uui-core';
-import { CheckboxGroup as uuiCheckboxGroup, CheckboxGroupProps } from '@epam/uui-components';
-import { Checkbox } from '../inputs/Checkbox';
+import * as React from 'react';
+import { cx, directionMode, ICanBeInvalid, ICanBeReadonly, IDisableable, IEditable, IHasCX, IHasDirection, IHasForwardedRef, IHasRawProps } from '@epam/uui-core';
+import { Checkbox, CheckboxProps } from '../inputs';
 import css from './CheckboxGroup.module.scss';
 
-export const CheckboxGroup = withMods<CheckboxGroupProps<any>>(
-    uuiCheckboxGroup,
-    () => [css.root],
-    () => ({ CheckboxInput: Checkbox }),
-);
+interface CheckboxGroupItem<TValue> extends Omit<CheckboxProps, 'id' | 'onValueChange' | 'value'> {
+    /** Checkbox label. Can be a string, or React.Element. */
+    name: React.ReactNode;
+    /** Item ID to put into selection */
+    id: TValue;
+    /** Render callback for checkbox label.
+     * If omitted, 'name' prop value will be rendered.
+     */
+    renderName?: () => React.ReactNode;
+}
+
+export interface CheckboxGroupProps<TValue>
+    extends ICanBeInvalid,
+    IHasCX,
+    IEditable<TValue[]>,
+    IDisableable,
+    IHasDirection,
+    ICanBeReadonly,
+    IHasRawProps<React.FieldsetHTMLAttributes<HTMLFieldSetElement>>,
+    IHasForwardedRef<HTMLFieldSetElement> {
+    /** Overrides the component to render a single checkbox */
+    CheckboxInput?: React.ComponentType<CheckboxProps>;
```

Comment:
Now we don't need CheckboxInput prop, since we are exporting it from uui package

Reply:
fixed